virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -954,33 +956,33 @@ You can view the status conditions and volume snapshot status on the *Diagnostic
 |Description
 
 |*Status conditions* table
-|Display a list of conditions that are reported for all aspects of a VM.
+|Display a list of conditions that are reported for the virtual machine.
 
 |*Filter* field
 |Filter status conditions by category and condition.
 
-|*Search* field
+|Search field
 |Search status conditions by reason.
 
 |*Manage columns* icon
-|Select columns to display.
+|Select the columns to display.
 
-|*Volume snapshot* table
-|List of volumes, their snapshot enablement status, and reason
+|*Volume snapshot status* table
+|List of volumes, their snapshot enablement status, and reason.
 |====
 =====
 
-[id="ui-templates-page"]
+[id="templates-page_virt-web-console-overview"]
 == Templates page
 
-You can create, edit, and clone VirtualMachine templates on the Templates page.
+You create, edit, and clone virtual machine templates on the *Templates* page.
 
 [NOTE]
 ====
-You cannot edit a Red Hat template. You can clone a Red Hat template and edit it to create a custom template.
+You cannot edit a Red Hat template. You must clone a Red Hat template and edit it to create a custom template.
```

Comment:
I see where you were going with this, but it still seems a little confusing to me. Maybe "However, you can clone a Red Hat template and edit it to create a custom template."